src/aig-common/modules/standard/components/ipp-modality/ipp-modality-form/ipp-modality-form.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { FuseProgressBarService } from '@fuse/components/progress-bar/progress-bar.service';
+import { ItalianPublicProcurementModalityResourceService, ItalianPublicProcurementModalityDTO  } from 'aig-standard';
+import { EventService } from 'aig-common/event-manager/event.service';
+
+@Component({
+    selector: 'aig-ipp-modality-form',
+    templateUrl: './ipp-modality-form.component.html',
+    styleUrls: ['./ipp-modality-form.component.scss']
+})
+export class AigIppModalityFormComponent implements OnInit {
```

Comment:
name of component not correct. Correct name:
`AigIppModalityNewUpdateFormComponent`
fix also the name of files.